src/sonic-yang-models/tests/yang_model_tests/tests/vlan.json

```diff
@@ -19,7 +19,14 @@
     },
     "VLAN_WITH_NON_EXIST_PORT": {
 	"desc": "Configure a member port in VLAN_MEMBER table which does not exist.",
-	"eStrKey" : "LeafRef"
+	"eStrKey" : "InvalidValue"
```

Comment:
Must be LeafRef only?,

Reply:
The type of vlan member (ifname) is changed from "leafref" to a union of 2 leafrefs (port and portchannel). Hence an "InvalidValue" error is returned from libyang. 

In the existing TC (with LeafRef), seeing this build error:
------------------- Test 59: Configure a member port in VLAN_MEMBER table which does not exist.---------------------
libyang[0]: Invalid value "Ethernet156" in "ifname" element. (path: /sonic-vlan:sonic-vlan/VLAN_MEMBER/VLAN_MEMBER_LIST[name='Vlan100'][ifname='Ethernet156']/ifname)
INFO:YANG-TEST:Invalid value "Ethernet156" in "ifname" element.
**ERROR:YANG-TEST: Exception >Mismatch ['Leafref', 'non-existing'] and Invalid value "Ethernet156" in "ifname" element.< in /sonic/src/sonic-yang-models/tests/yang_model_tests/test_yang_model.py:211**
INFO:YANG-TEST:Configure a member port in VLAN_MEMBER table which does not exist. Failed

Since the type is changed, I've modified the "eStrKey" appropriately.

Reply:
okie, sure if Libyang behaves this way...they we can use Invalid value only